packages/hoppscotch-app/components/collections/my/Request.vue

```diff
@@ -362,6 +362,9 @@ const selectRequest = () => {
       setRESTSaveContext(null)
     }
   }
+  if (getRESTRequest().endpoint === getDefaultRESTRequest().endpoint) {
```

Comment:
This only checks for the same endpoints, why shouldn't we just check against the entire default request itself ?